Mathlib/AlgebraicTopology/SimplicialSet/SubcomplexColimits.lean

```diff
@@ -0,0 +1,104 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+module
+
+public import Mathlib.AlgebraicTopology.SimplicialSet.Subcomplex
+public import Mathlib.CategoryTheory.Limits.Shapes.Pullback.CommSq
+public import Mathlib.CategoryTheory.Limits.Types.Multicoequalizer
+public import Mathlib.CategoryTheory.Limits.Types.Pushouts
+
+/-!
+# Colimits involving subcomplexes of a simplicial set
+
+If `X` is a simplicial set, and we have subcomplexes `A`, `U i` (for `i : ι`) and
+`V i j` which satisfy `Subcomplex.MulticoequalizerDiagram A U V` (an abbreviation
+for `CompleteLattice.MulticoequalizerDiagram`), we
+show that the simplicial sset corresponding to `A` is the multicoequalizer of
+the `U i` along the `V i j`.
+
+Similarly, bicartesian squares in the lattice `Subcomplex X` give pushout
+squares in the category of simplicial sets.
+
+-/
+
+@[expose] public section
+
+universe u
+
+open CategoryTheory Limits
+
+namespace SSet
+
+namespace Subcomplex
+
+variable {X : SSet.{u}}
+
+section
+
+variable {A : X.Subcomplex} {ι : Type*}
+  {U : ι → X.Subcomplex} {V : ι → ι → X.Subcomplex}
+
+variable (A U V) in
+/-- Abbreviation for multicoequalizer diagrams in the complete lattice of
+subcomplexes of a simplicial set. -/
+abbrev MulticoequalizerDiagram := CompleteLattice.MulticoequalizerDiagram A U V
```

Comment:
This made me look in the file Order/CompleteLattice/MulticoequalizerDiagram.lean, and I noticed the fields of the structures defined there which involve `⊔`/`⊓` are named `max_eq`/`min_eq`. Shouldn't it be `sup_eq`/`inf_eq`? Also, in `CompleteLattice.MulticoequalizerDiagram`, the `min_eq` field should be named `eq_min` if it is written in that direction (or `eq_inf` if we change from the `max`/`min` naming).

Reply:
Thanks for the suggestion! I have changed the names.